src/index.html

```diff
@@ -1,14 +1,34 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+      >
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">
+        <img src="images/logo.png" alt="Логотип">
+      </div>
+      <nav class="container">
+        <ul class="list">
+          <li class="item "><a class="is-active" href="#">Apple</a></li>
+          <li class="item"><a href="#">Samsung</a></li>
+          <li class="item"><a href="#">Смартфоны и телефоны</a></li>
+          <li class="item" data-qa="hover"><a href="#">Ноутбуки и компьютеры</a></li>
+          <li class="item"><a href="#">Гаджеты</a></li>
+          <li class="item"><a href="#">Планшеты</a></li>
+          <li class="item"><a href="#">Фото</a></li>
+          <li class="item"><a href="#">Видео</a></li>
```

Comment:
слишком длинные строки получаются, давай переносить
```suggestion
          <li class="item">
            <a class="is-active" href="#">Apple</a>
          </li>
          <li class="item"><a href="#">Samsung</a></li>
          <li class="item"><a href="#">Смартфоны и телефоны</a></li>
          <li class="item" data-qa="hover"><a href="#">Ноутбуки и компьютеры</a></li>
          <li class="item"><a href="#">Гаджеты</a></li>
          <li class="item"><a href="#">Планшеты</a></li>
          <li class="item"><a href="#">Фото</a></li>
          <li class="item"><a href="#">Видео</a></li>
```